cv32e40s/tests/cfg/pmp.yaml

```diff
@@ -5,169 +5,6 @@ compile_flags: >
     +define+ZBA_ZBB_ZBC_ZBS
 ovpsim: >
     --override cpu/PMP_registers=64
-    --override cpu/PMP_undefined=T
```

Comment:
Are these still present in the other configs?

Reply:
Yes. I have not edited the other yamls.